pkgs/applications/networking/cluster/terraform-providers/providers.json

```diff
@@ -824,9 +826,11 @@
   },
   "segment": {
     "owner": "ajbosco",
+    "provider-source-address_no_registry": null,
```

Comment:
Skipping the update script here as well, or we could move it to a separate derivation or remove it entirely.

Reply:
Let's remove it.